### ISSUE TITLE
Visible and hidden OpenBikeSensor readings

### ISSUE THREAD

**Issue by selphiron** [simra-project/simra-ios#72]
When a user presses the hardware button of the OpenBikeSensor connected to the SimRa App via Bluetooth, the following should happen:

At the nearest GPS position of the SimRa ride, a near miss incident with the incident type "Close Pass" should be created with the following description:
```
OpenBikeSensor
l: <OpenBikeSensor left sensor value in cm>
r: <OpenBikeSensor right sensor value in cm>
```
Please mind the line breaks after OpenBikeSensor and <OpenBikeSensor left sensor value in cm>.

If either the right or left sensor readings are between -100cm and 150 cm (inclusive in both cases), the incident should be visible to the user and be uploaded, whether the user annotates that incident or not. Such incidents start with a key value of 2000 (,2001, 2002, ...). Their incident type is 1.

Otherwise, the incident should still be created and uploaded but not shown to the user as an incident. Such incidents start with a key value of 3000 (,3001, 3002, ...). Their incident type is -2.

**Comment by ckrey** [comment]
For clarification:

> When a user presses the hardware button of the OpenBikeSensor connected to the SimRa App via Bluetooth, the following should happen:
At the nearest GPS position of the SimRa ride, a near miss incident with the incident type "Close Pass" should be created 

- [x] already existing


> with the following description:

- [ ] needs to change

> If either the right or left sensor readings are between -100cm and 150 cm (inclusive in both cases), the incident should be visible to the user and be uploaded, whether the user annotates that incident or not.

- [ ] currently valid sensor readings are defined as if left or right reading < 150cm

> Such incidents start with a key value of 2000 (,2001, 2002, ...). Their incident type is 1.

- [ ] currently all incidents are numbered in one scheme 0(,1, 2, ...)

> Otherwise, the incident should still be created and uploaded but not shown to the user as an incident. Such incidents start with a key value of 3000 (,3001, 3002, ...). Their incident type is -2.

- [ ] currently in such a case no incident is created

**Comment by selphiron** [comment]
Yes, these changes should be done, so that the Android and iOS versions behave the same when combined with an OBS

**Comment by ckrey** [comment]
So the key sequence might be:
- 0
- 1
- 2
- 2000
- 3
- 3000
- 3001
- 4
- 5
- 2001

?

**Comment by ckrey** [comment]
Problem is I will have make the changes without a chance to test (no OBS here)

**Comment by selphiron** [comment]
no, they should be ordered, so

0
1
2
3
4
5
2000
2001
3000
3001

sorry for the very late answer
